app/assets/javascripts/modules/track-qa-choices.js

```diff
@@ -0,0 +1,30 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (global, GOVUK) {
+  'use strict'
+
+  var $ = global.jQuery
+
+  GOVUK.Modules.TrackQaChoices = function () {
+    this.start = function (element) {
```

Comment:
Can `element` be `$element` here to show it's jQuery-wrapped?

Reply:
:+1: I prefer that convention

Reply:
https://github.com/alphagov/finder-frontend/pull/678/commits/bed2ad90c7a98d84aa7f8fe010043910b57120e8#diff-e1ff46625134b9f4a5a8424b263484a0R9

Reply:
👍